dbt/models/sec10k/core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities/schema.yml

```diff
@@ -5,13 +5,10 @@ sources:
       - name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
         data_tests:
           - expect_columns_not_all_null
-          - dbt_expectations.expect_table_row_count_to_be_between:
-              description: Nightly builds and local ETL yield slightly different output
-              config:
-                enabled: "{{ target.name == 'etl-full'}}"
+          - check_row_counts_per_partition:
               arguments:
-                min_value: 5543
-                max_value: 5546
+                table_name: core_sec10k__assn_exhibit_21_subsidiaries_and_eia_utilities
+                partition_expr:
```

Comment:
Maybe I'm being overly optimistic, but I'm getting the same row-counts locally and in the nightly builds now, so let's try getting rid of this weird special case. Maybe some bugfix in pandas or another library we're using made something platform-dependent platform-independent... 🤞🏼